app-5-rdf-serializer/src/main/java/org/geovistory/toolbox/streams/rdf/I.java

```diff
@@ -0,0 +1,16 @@
+package org.geovistory.toolbox.streams.rdf;
+
+public enum I {
```

Comment:
This class seems to be used by tests only. I suggest to move it into the test folder.